ApplicationSettings_client.py

```diff
@@ -0,0 +1,519 @@
+"""
+application_settings.py
+
+This software is released under the AGPL-3.0 license
+Copyright (c) 2023-2024 Braedon Hendy
+
+Further updates and packaging added in 2024 through the ClinicianFOCUS initiative, 
+a collaboration with Dr. Braedon Hendy and Conestoga College Institute of Applied 
+Learning and Technology as part of the CNERG+ applied research project, 
+Unburdening Primary Healthcare: An Open-Source AI Clinician Partner Platform". 
+Prof. Michael Yingbull (PI), Dr. Braedon Hendy (Partner), 
+and Research Students - Software Developer Alex Simko, Pemba Sherpa (F24), and Naitik Patel.
+
+This module contains the ApplicationSettings class, which manages the settings for an
+application that involves audio processing and external API interactions, including
+KoboldCPP, WhisperAudio, and OpenAI services.
+
+"""
+
+import json
+import tkinter as tk
+from tkinter import ttk, messagebox
+
+
+class ApplicationSettings:
```

Comment:
It should be okay for now right?! Do you want me to start refactoring like sourcery suggested? If so, I think it might be helpful for me to redo the directory structure in another PR as well.

Reply:
The bot speaks truth.

Reply:
Good for now, but fair for future.